src/example/p4p5/profiles/uninstall-plone4/cssregistry.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```

Comment:
sure? no need for cssregistry.xml and jsregistry.xml in uninstall profiles? thats new for me...